hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestIncrementalBackup.java

```diff
@@ -395,6 +395,11 @@ public void TestIncBackupRestoreWithOriginalSplitsSeperateFs() throws Exception
       assertTrue(checkSucceeded(fullBackupId));
 
       insertIntoTable(conn, table1, famName, 4, 100);
+
+      HRegion regionToBulkload = TEST_UTIL.getHBaseCluster().getRegions(table1).get(0);
```

Comment:
This new test fails on master but is fixed by your correction? Nice one, these are annoying to track down.